internal/graph/graph.go

```diff
@@ -420,6 +421,33 @@ func newTree(prof *profile.Profile, o *Options) (g *Graph) {
 	return selectNodesForGraph(nodes, o.DropNegative)
 }
 
+var javaFullRegExp = regexp.MustCompile(`^(?:[a-z]\w*\.)*[A-Z][\w\$]*\.(?:<init>|[a-z]\w*(?:\$\d+)?)(?:\(|$)`)
+var javaShortRegExp = regexp.MustCompile(`([A-Z][\w\$]*\.(?:<init>|[a-z]\w*(?:\$\d+)?))(?:|$)`)
+var goPrefixRegExp = regexp.MustCompile(`^(?:[a-z]\w*(\.\w*)*\/)+.`)
+var goPathRegExp = regexp.MustCompile(`^(?:[a-z]\w*(\.\w*)*\/)+`)
+var cppFullRegExp = regexp.MustCompile(`^(?:[_a-zA-Z]\w*\::)*_*[A-Z]\w*::~?[_a-zA-Z]\w*$`)
+var cppAnonymousPrefixRegExp = regexp.MustCompile(`^\(anonymous namespace\)::`)
+
+// GetNodeShortName returns a shortened version of a function's name.
+func GetNodeShortName(f string) string {
```

Comment:
Maybe ShortenFunctionName ? The comment describing this function talks about functions, not nodes.

Reply:
Done.